### PR TITLE
fix(config): allow triple-slash directives on .d.ts files

### PR DESCRIPTION
**Pull request by deini**
@chanceaclark Noticed an issue with 3ple-slash directives in `d.ts` files: 

image

### CODE REVIEW

**Review by chanceaclark — APPROVED** [review]
🙏

File: packages/eslint-config/test/eslint.spec.js [hunk 1]
```diff
@@ -38,6 +38,12 @@ it('keeps rules stable TSX', async () => {
   expect(output).toMatchSnapshot();
 });
 
+it('keeps rules stable *.d.ts', async () => {
+  const output = await getEslint().calculateConfigForFile(join(__dirname, 'file.tsx'));
```
Comment (icatalina) [inline packages/eslint-config/test/eslint.spec.js:42]: Shouldn't this use `.d.ts` as filename?

**Review by icatalina — COMMENTED** [review]
@deini just realised this might be wrong... 😕

File: packages/eslint-config/test/eslint.spec.js (hunk 1 above)
Comment (deini) [reply packages/eslint-config/test/eslint.spec.js:42]: lol, yes.

**Pull request merged**